README.md

```diff
@@ -107,6 +107,10 @@ LOGGING_LEVEL is INFO by default. Allowed logging levels are NOTSET, DEBUG, INFO
 
 ```bash
 conda create -n rag-test python=3.11
```

Comment:
I would challenge why we even need conda. . .

Reply:
you mean, because we can run it in a dev container?
https://github.com/microsoft/rag-experiment-accelerator/issues/334#issuecomment-2007342981

Reply:
> Yes - and what do we get from conda that this repo needs?

It provides isolation, and could prevent clashing with other projects.
I agree that if we use dev container then it's redundant, but some developers prefer to work without a container